### DISCUSSION TITLE
Lighthouse deck photodiode loose

### DISCUSSION THREAD

**Discussion by lbult** [bitcraze/discussions#763] [Q&A]
Hi,

I have two Lighthouse decks where one of the reflectors got loose from one of its two (soldered?) PCB/deck connection. Is it possible to just resolder this connection? If not, is there a different way to repair it?

Thanks in advance!

**Comment by tobbeanton** [comment]
Yes, you can most likely just resolder the connections. If the photodiode is broken you can get new ones. It's the [BPW 34 S-Z](https://www.findchips.com/search/BPW%2034%20S-Z)
